packages/contracts/coverage.sh

```diff
@@ -20,8 +20,16 @@ lcov \
     --rc lcov_branch_coverage=1 \
     --remove merged-lcov.info \
     --output-file filtered-lcov.info \
-    "*node_modules*" "*test*" "*mock*" "*scripts*"
-
+    --ignore-errors unused \
+    "*node_modules*" \
+    "*test*" \
+    "*mock*" \
+    "*scripts*" \
+    "src/dollar/libraries/*" \
```

Comment:
Again, pls remove the `src/dollar/libraries/*` path as it should be processed by the coverage script

Reply:
ok, changed as well